exercises/practice/largest-series-product/.docs/instructions.md

```diff
@@ -0,0 +1,14 @@
+# Instructions
+
+Given a string of digits, calculate the largest product for a contiguous substring of digits of length n.
+
+For example, for the input `'1027839564'`, the largest product for a series of 3 digits is 270 `(9 * 5 * 6)`, and the largest product for a series of 5 digits is 7560 `(7 * 8 * 3 * 9 * 5)`.
+
+Note that these series are only required to occupy *adjacent positions* in the input; the digits need not be *numerically consecutive*.
+
+For the input `'73167176531330624919225119674426574742355349194934'`,
+the largest product for a series of 6 digits is 23520.
+
+For a series of zero digits, the largest product is 1 because 1 is the multiplicative identity.
+(You don't need to know what a multiplicative identity is to solve this problem;
+it just means that multiplying a number by 1 gives you the same number.)
```

Comment:
It's a bit confusing that we say they don't need to know and then explain it (kinda incorrectly, because the concept isn't just for numbers).

For me at least, it was initially confusing that the product of no numbers is 1 (or why any([]) == false and all([]) == true). I think the description should explain why we do this (so prod([]) * prod(v) == prod(v) because v == [[]..., v...]).

Reply:
Would you mind opening a PR on problem-specifications? (I would be hard pressed to phrase it well)

Reply:
Sure. https://github.com/exercism/problem-specifications/pull/2234